tools/pythonpkg/tests/fast/arrow/test_integration.py

```diff
@@ -101,6 +103,41 @@ def test_decimals_roundtrip(self,duckdb_cursor):
         result = duckdb_conn.execute('select * from bigdecimal')
         assert (result.fetchone()[0] == 9999999999999999999999999999999999)
 
+    def test_intervals_roundtrip(self,duckdb_cursor):
```

Comment:
Can you also add a test where the interval is used in a nested structure?

Reply:
And tests for boundary values? (i.e., the min and max value supported by arrow in the interval type)

Reply:
And null values

Reply:
I updated test for above